mailerlite/client.py

```diff
@@ -90,7 +90,7 @@ def make_request(url, method, headers=None, data=None,
         raise ValueError("Incorrect request method. method should be "
                          "{}".format(VALID_REQUEST_METHODS))
 
-    url = pjoin(MAILERLITE_API_V2_URL, url)
+    url = urljoin(MAILERLITE_API_V2_URL, url)
```

Comment:
Good Catch! I need to add Windows CI's via GitHub action to catch this kind of error.

Reply:
Also just removed the import that wasn't needed any more